test/functionalTest/cases/947_v2_entities/GET_v2_entities.test

```diff
@@ -0,0 +1,426 @@
+            }
+        }
+    ]
+}
+
+
+03. GET /v2/entities
+====================
+HTTP/1.1 200 OK
+Content-Length: 232
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    {
+        "A0": 255, 
+        "A1": "E1/T1/A1", 
+        "A2": {
+            "type": "AT2", 
+            "value": "E1/T1/A2"
+        }, 
+        "A3": {
+            "m1": "M1", 
+            "type": "a", 
+            "value": "E1/T1/A3/M1"
+        }, 
+        "A4": {
+            "m1": "M1", 
+            "m2": {
+                "type": "tm2", 
+                "value": "M2"
+            }, 
+            "m3": 3, 
+            "type": "a4", 
+            "value": "E1/T1/A4/M1-3"
+        }, 
+        "id": "E1", 
+        "type": "T1"
+    }
+]
+
+
+04. GET /v2/entities?count=true
+===============================
+HTTP/1.1 200 OK
+Content-Length: 232
+Content-Type: application/json
+X-Total-Count: 1
+Date: REGEX(.*)
+
+[
+    {
+        "A0": 255, 
+        "A1": "E1/T1/A1", 
+        "A2": {
+            "type": "AT2", 
+            "value": "E1/T1/A2"
+        }, 
+        "A3": {
+            "m1": "M1", 
+            "type": "a", 
+            "value": "E1/T1/A3/M1"
+        }, 
+        "A4": {
+            "m1": "M1", 
+            "m2": {
+                "type": "tm2", 
+                "value": "M2"
+            }, 
+            "m3": 3, 
+            "type": "a4", 
+            "value": "E1/T1/A4/M1-3"
+        }, 
+        "id": "E1", 
+        "type": "T1"
+    }
+]
+
+
+05. POST /v1/updateContext with compound attribute value
+========================================================
+HTTP/1.1 200 OK
+Content-Length: 377
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A0", 
+                        "type": "", 
+                        "value": ""
+                    }
+                ], 
+                "id": "E2", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+06. GET /v2/entities
+====================
+HTTP/1.1 200 OK
+Content-Length: 287
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    {
+        "A0": 255, 
+        "A1": "E1/T1/A1", 
+        "A2": {
+            "type": "AT2", 
+            "value": "E1/T1/A2"
+        }, 
+        "A3": {
+            "m1": "M1", 
+            "type": "a", 
+            "value": "E1/T1/A3/M1"
+        }, 
+        "A4": {
+            "m1": "M1", 
+            "m2": {
+                "type": "tm2", 
+                "value": "M2"
+            }, 
+            "m3": 3, 
+            "type": "a4", 
+            "value": "E1/T1/A4/M1-3"
+        }, 
+        "id": "E1", 
+        "type": "T1"
+    }, 
+    {
+        "A0": {
```

Comment:
What happen if one of the fields of the compound is named `value`, `type` or `metadata` ?